polybar-scripts/updates-void/updates-void.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# If `xbps-install -Sun` returns an error saying it failed to initialize libxbps
+# you may need to add this script to your sudoers file so you can run it with sudo
+
+updates=$(/usr/bin/xbps-install -Sun 2> /dev/null | wc -l)
+
+if [[ -z "$updates" ]] || [[ "$updates" -eq '0' ]]; then
+    exit 1
```

Comment:
Exit the script somewhere in your code is not really clean. Try to avoid the `exit` command here.
Look at the Arch update script, I guess you could copy the whole script. You need just to change the command.